src/Components/ErrorPage/ErrorPage.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react'
-
+import { Link } from 'react-router-dom';
+import './ErrorPage.css'
 
 const ErrorPage = () => {
     return (
-        <p>
-            Error!
+        <div className="error-container">
+        <p className="error-string">
+            404 ERROR. SERVER ERROR... PLEASE TRY AGAIN LATER
```

Comment:
You could maybe make this message more empathetic to the user. Not all users will know what this means. Maybe something like `Uh Oh! Looks like something went wrong on our end. Please try again later!` and then you can also display the 404 message as well for those who do know what it is. Just a suggestion, though!